packages/taquito/src/context.ts

```diff
@@ -32,15 +33,17 @@ export interface Config {
   defaultConfirmationCount?: number;
   shouldObservableSubscriptionRetry?: boolean;
   observableSubscriptionRetryDelay?: number;
-  observableSubscriptionRetries?: number;
+  observableSubscriptionMaxRetryDelay?: number;
+  observableSubscriptionRetryFunction?: Function;
 }
 
 export const defaultConfig: Partial<Config> = {
   defaultConfirmationCount: 1,
   confirmationPollingTimeoutSecond: 180,
   shouldObservableSubscriptionRetry: false,
-  observableSubscriptionRetries: 10,
-  observableSubscriptionRetryDelay: 1000
+  observableSubscriptionRetryFunction: retry,
+  observableSubscriptionRetryDelay: 1000, // 1 minute
```

Comment:
comment says "1 minute", but it looks like it's actually 1 second? Perhaps it's better not to have a comment here at all, and either make units obvious from the name (like confirmationPollingTimeoutSecond does) or stick to the library conventions if applicable (maybe all timeouts/delays are consistently specified in the library in milliseconds ?)

Reply:
Fixed comment, thank you.